examples/CantileverDistributedLoad/cantilever_nonconservative_distributed_load.py

```diff
@@ -0,0 +1,250 @@
+from matplotlib import pyplot as plt
+import numpy as np
+import elastica as ea
+import json
+
+from cantilever_distrubuted_load_postprecessing import (
+    plot_video_with_surface,
+    find_tip_position,
+    adjust_square_cross_section,
+    NonconserativeForce,
+)
+
+
+def cantilever_subjected_to_a_nonconservative_load(
+    n_elem,
+    base_length,
+    side_length,
+    base_radius,
+    youngs_modulus,
+    dimentionless_varible,
+    Animation=False,
+    PLOT_FIGURE_Equilibrium=False,
+):
+    class StretchingBeamSimulator(
+        ea.BaseSystemCollection, ea.Constraints, ea.Forcing, ea.Damping, ea.CallBacks
+    ):
+        pass
+
+    stretch_sim = StretchingBeamSimulator()
+
+    stretchable_rod = ea.CosseratRod.straight_rod(
+        n_elem,
+        start,
+        direction,
+        normal,
+        base_length,
+        base_radius,
+        density,
+        youngs_modulus=youngs_modulus,
+        shear_modulus=shear_modulus,
+    )
+
+    adjust_square_cross_section(stretchable_rod, youngs_modulus, side_length)
+
+    stretch_sim.append(stretchable_rod)
+
+    stretch_sim.constrain(stretchable_rod).using(
+        ea.OneEndFixedBC, constrained_position_idx=(0,), constrained_director_idx=(0,)
+    )
+
+    load = (youngs_modulus * I * dimentionless_varible) / (
+        density * base_area * (base_length**3)
+    )
+
+    stretch_sim.add_forcing_to(stretchable_rod).using(NonconserativeForce, load)
+
+    # add damping
+    dl = base_length / n_elem
+    dt = 0.1 * dl / 50
+    damping_constant = 0.2
+
+    stretch_sim.dampen(stretchable_rod).using(
+        ea.AnalyticalLinearDamper,
+        damping_constant=damping_constant,
+        time_step=dt,
+    )
+
+    # Add call backs
+    class AxialStretchingCallBack(ea.CallBackBaseClass):
```

Comment:
Change callback name to this case

Reply:
Did you change this? It's still `AxialStretchingCallBack`. Can you change it to something like `NonConservativeDistributedLoadCallBack`?